tt_metal/hw/inc/wormhole/eth_l1_address_map.h

```diff
@@ -67,11 +52,29 @@ struct address_map {
   static constexpr uint32_t ISSUE_CQ_CB_BASE = SEMAPHORE_BASE + SEMAPHORE_SIZE;  // SIZE from shared common addr
   static constexpr uint32_t COMPLETION_CQ_CB_BASE = ISSUE_CQ_CB_BASE + 7 * L1_ALIGNMENT;
 
-  static constexpr std::int32_t ERISC_L1_ARG_BASE = COMPLETION_CQ_CB_BASE + 7 * L1_ALIGNMENT;
+  // Extra 4 bytes is to make sure that the launch message is properly aligned.
+  static constexpr std::int32_t ERISC_MEM_MAILBOX_BASE = COMPLETION_CQ_CB_BASE + 7 * L1_ALIGNMENT + 4;
+  // erisc early exit functionality re-uses mailboxes_t::ncrisc_halt_msg_t::stack_save memory
+  static constexpr std::int32_t ERISC_MEM_MAILBOX_STACK_SAVE = ERISC_MEM_MAILBOX_BASE + 4;
+
+  static constexpr std::int32_t ERISC_RING_BUFFER_ADDR = ERISC_MEM_MAILBOX_BASE + 128 + 12;
+  static constexpr std::int32_t PRINT_BUFFER_ER = ERISC_RING_BUFFER_ADDR + 128;
```

Comment:
Are these alignment increments? Or if they are sizes, try to grab them from `common_runtime_address_map.h`

Reply:
All of these hardcoded values/offsets are basically copied from what they were before I moved them, so I haven't really added anything new/changed anything. I'll take a look if they were defined anywhere.